### ISSUE TITLE
Loss function and the pretrained model (DSM)

### ISSUE THREAD

**Issue by ldz7** [autonlab/auton-survival#66]
Thank you for this wonderful job. 

I have some questions about the loss function. 

According to the original paper of DSM (Deep Survival Machines: Fully Parametric Survival Regression and Representation Learning for Censored Data with Competing Risks. IEEE Journal of Biomedical & Health Informatics (2021)), the loss function consists of three parts, i.e., uncensored, censoring and prior. 

I have found the uncensored loss and censoring loss in the code `losses.py`, but I haven't saw anything related to the prior loss. In the code https://github.com/autonlab/auton-survival/blob/eb88f7969323237633dc307dd86cfb9b78195a74/auton_survival/models/dsm/utilities.py#L132, I noticed that during the training of DSM model, a pretrained model is trained first to fill the shape and scale value of DSM. (I think these two parameters correspond to $\tilde{\beta}_k$ and $\tilde{\eta}_k$). And this pre training process is not mentioned in the paper.

So my questions are:
1. Is the prior loss implemented in the code? If so, which part of code is about the prior loss?

2. Is the pretrained model related to the prior loss? (Because in the generative story of paper, it says "the set of parameters $\{\tilde{\beta}_k\}_{k=1}^{K}$ and $\{\tilde{\eta}_k\}_{k=1}^{K}$ are drawn from the **prior** $\beta_0$ and $\eta_0$.

3. (This one is not about the loss) Why does $\mathoperator{log} \tilde{\beta}_k$ follow $N (\beta_0, 1/\lambda)$? Could you give me some hints?

Thank you very much.
